locksmith/src/routes/v2/lock-settings.ts

```diff
@@ -23,11 +23,6 @@ router.get(
   getSettings
 )
 
-router.get(
-  '/slug/:slug',
-  authenticatedMiddleware,
-  lockManagerMiddleware,
-  getLockSettingsBySlug
-)
+router.get('/slug/:slug', getLockSettingsBySlug)
```

Comment:
no need of middleware, in order to allow also to non-authenticated user to see a specific page with slug

Reply:
Was there any reason before to add this? Just wondering if missed something here.

Reply:
Just to make it usable by the lock manager, but this is not right, and as we just retrieved the date here